docs/user/example-cases/harvey-flooding/critical-outflow-bc/index.md

```diff
@@ -0,0 +1,92 @@
+# Hurricane Harvey Flooding using MRMS Dataset and Critical Outflow BC
+
+The 1 km Multi-Radar Multi-Sensor System (MRMS) rainfall dataset is used
+in this simulation. The hourly MRMS dataset files
+are PETSc Vec that have been saved in the binary format and are named as
```

Comment:
You could say "Each hourly MRMS dataset file is a PETSc Vec saved in the binary format, named as ..."

I usually find it easier to talk about singular things rather than plural.

Reply:
updated